docs/content/release-notes/flink-1.15.md

```diff
@@ -0,0 +1,597 @@
+formats which led to inconsistent implementations. The `FileSystemTableSource` 
+has been updated to distinguish between those two interfaces now. Users that 
+implement custom formats for `FileSystemTableSource` might need to verify the 
+implementation and make sure to implement `ProjectableDecodingFormat` if necessary.
+
+#### Push down partitions before filters 
+
+##### [FLINK-24717](https://issues.apache.org/jira/browse/FLINK-24717)
+
+This might have an impact on existing table source implementations as push down 
+filters might not contain partition predicates anymore. However, the connector 
+implementation for table sources that implement both partition and filter push 
+down became easier with this change.
+
+#### Flink SQL `SUM()` causes a precision error 
+
+##### [FLINK-24691](https://issues.apache.org/jira/browse/FLINK-24691)
+
+This changes the result of a decimal `SUM()` between 1.14.0 and 1.14.1. It restores 
+the behavior of 1.13 to be consistent with Hive/Spark.
+
+#### Use the new casting rules in TableResult#print 
+
+##### [FLINK-24685](https://issues.apache.org/jira/browse/FLINK-24685)
+
+The string representation of `BOOLEAN` columns from DDL results 
+(`true/false -> TRUE/FALSE`), and row columns in DQL results 
+(`+I[...] -> (...)`) has changed for printing.
+
+#### Casting from a string to a DATE and TIME allows incomplete strings 
+
+##### [FLINK-24421](https://issues.apache.org/jira/browse/FLINK-24421)
+
+The defaults for casting incomplete strings like `"12"` to TIME have changed from `12:01:01` to `12:00:00`.
+
+#### Casting from STRING to TIMESTAMP_LTZ looses fractional seconds 
+
+##### [FLINK-24446](https://issues.apache.org/jira/browse/FLINK-24446)
+
+`STRING` to `TIMESTAMP(_LTZ)` casting now considers fractional seconds.
+Previously fractional seconds of any precision were ignored.
+
+#### Sinks built with the unified sink framework do not receive timestamps when used in Table API 
+
+##### [FLINK-24608](https://issues.apache.org/jira/browse/FLINK-Table)
+
+This adds an additional operator to the topology if the new sink interfaces are used 
+(e.g. for Kafka). It could cause issues in 1.14.1 when restoring from a 1.14 savepoint. 
+A workaround is to cast the time attribute to a regular timestamp in the SQL statement
+closely before the sink.
+
+#### SQL functions should return `STRING` instead of `VARCHAR(2000)` 
+
+##### [FLINK-24586](https://issues.apache.org/jira/browse/FLINK-24586)
+
+Functions that returned `VARCHAR(2000)` in 1.14, return `VARCHAR` with maximum 
+length now. In particular this includes:
+
+```SQL
+SON_VALUE
+CHR
+REVERSE
+SPLIT_INDEX
+REGEXP_EXTRACT
+PARSE_URL
+FROM_UNIXTIME
+DECODE
+DATE_FORMAT
+CONVERT_TZ
+```
+
+#### Support IS JSON for Table API 
+
+##### [FLINK-16501](https://issues.apache.org/jira/browse/FLINK-16501)
+
+`IS JSON` does not return `NULL` anymore but always `FALSE` (even if the argument is `NULL`).
```

Comment:
the ticket title and description don't really fit. The title makes it seem sound like a new feature, while the description seems more like a behavior change.